node.gyp

```diff
@@ -492,40 +492,7 @@
           'product_name': 'node_base',
         }],
         [ 'v8_enable_inspector==1', {
-          'defines': [
-            'HAVE_INSPECTOR=1',
-          ],
-          'sources': [
-            'src/inspector_agent.cc',
-            'src/inspector_io.cc',
-            'src/inspector_js_api.cc',
-            'src/inspector_socket.cc',
-            'src/inspector_socket_server.cc',
-            'src/inspector/main_thread_interface.cc',
-            'src/inspector/worker_inspector.cc',
-            'src/inspector/node_string.cc',
-            'src/inspector/worker_agent.cc',
-            'src/inspector/tracing_agent.cc',
-            'src/inspector_agent.h',
-            'src/inspector_io.h',
-            'src/inspector_socket.h',
-            'src/inspector_socket_server.h',
-            'src/inspector/main_thread_interface.h',
-            'src/inspector/worker_inspector.h',
-            'src/inspector/node_string.h',
-            'src/inspector/worker_agent.h',
-            'src/inspector/tracing_agent.h',
-            '<@(node_inspector_generated_sources)'
-          ],
-          'dependencies': [
-            'node_protocol_generated_sources#host',
-            'v8_inspector_compress_protocol_json#host',
-          ],
-          'include_dirs': [
-            '<(SHARED_INTERMEDIATE_DIR)/include', # for inspector
-            '<(SHARED_INTERMEDIATE_DIR)',
-            '<(SHARED_INTERMEDIATE_DIR)/src', # for inspector
-          ],
+          'includes' : [ 'src/inspector/node_inspector.gypi' ],
```

Comment:
I think this would be more consistent if we also had e.g. the main sources list for `src/` in `src/node.gypi` or similar…

Reply:
That's high on my backlog.
Might also have a compile only target for `/src/` similar to `v8_base`, then do the linking in `node_lib`. So you can run compile with no dependencies by running `make -C out node_base`